webapp/loggedin/modules/users/user/components/profilePictureEditor.js

```diff
@@ -1,5 +1,7 @@
 import './profilePictureEditor.scss'
 
+import * as PropTypes from 'prop-types'
+
 import * as FileTypes from '@webapp/utils/fileTypes'
 
 import React, { useEffect, useRef, useState } from 'react'
```

Comment:
order should be as below and no need to import * from prop-types
```
import React, { useEffect, useRef, useState } from 'react'
import PropTypes from 'prop-types'

...rest
```

Reply:
ok, right